src/tools/translation-namespaces.ts

```diff
@@ -9,6 +9,7 @@ export const dashboardNamespaces = [
   'settings/general/internationalization',
   'settings/general/module-enabler',
   'settings/general/search-engine',
+  'settings/general/search-new-tab',
```

Comment:
You don't need to add a new translation namespace. Use the existing search module one instead.

Reply:
I moved it to settings/general/search-engine